### PR TITLE
Reorganized and refactored the code

### PR DESCRIPTION
**Pull request by reachjainrahul**
Also split accountPoller into accountmanager and accountpoller.

TODO: Add unit test for account manager and improve coverage for cpa and ces controller

### CODE REVIEW

File: pkg/accountmanager/manager.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
+// Copyright 2022 Antrea Authors.
```
Comment (reachjainrahul) [inline pkg/accountmanager/manager.go:1]: 2023

File: pkg/cloudprovider/cloudapi/azure/azure_crd_helpers.go [hunk 2]
```diff
@@ -15,6 +15,7 @@
 package azure
 
 import (
+	"antrea.io/nephe/pkg/labels"
```
Comment (reachjainrahul) [inline pkg/cloudprovider/cloudapi/azure/azure_crd_helpers.go:18]: sort

File: pkg/controllers/cloudentityselector/controller.go [hunk 3]
```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 Antrea Authors.
```
Comment (reachjainrahul) [inline pkg/controllers/cloudentityselector/controller.go:1]: 2023

File: pkg/controllers/cloudentityselector/controller_test.go [hunk 4]
```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Antrea Authors.
```
Comment (reachjainrahul) [inline pkg/controllers/cloudentityselector/controller_test.go:1]: 2023

File: pkg/converter/source/virtualmachine.go [hunk 5]
```diff
@@ -15,13 +15,14 @@
 package source
 
 import (
+	"antrea.io/nephe/pkg/config"
```
Comment (reachjainrahul) [inline pkg/converter/source/virtualmachine.go:18]: sort

File: pkg/inventory/store/vpc_inventory.go [hunk 6]
```diff
@@ -15,6 +15,8 @@
 package store
 
 import (
+	"antrea.io/nephe/pkg/inventory/indexer"
```
Comment (reachjainrahul) [inline pkg/inventory/store/vpc_inventory.go:18]: sort and labels2

File: cmd/nephe-controller/main.go [hunk 7]
```diff
@@ -95,43 +100,47 @@ func main() {
 	cloudInventory := inventory.InitInventory()
 
 	// Initialize Account poller map.
```
[the remark this answers is not in the document]
Comment (Anandkumar26) [reply cmd/nephe-controller/main.go]: Done

File: pkg/accountmanager/manager.go (hunk 1 above)
Comment (Anandkumar26) [reply pkg/accountmanager/manager.go]: Done

File: pkg/cloudprovider/cloudapi/azure/azure_crd_helpers.go (hunk 2 above)
Comment (Anandkumar26) [reply pkg/cloudprovider/cloudapi/azure/azure_crd_helpers.go:18]: Done

File: pkg/inventory/store/vm_inventory.go [hunk 8]
```diff
@@ -15,6 +15,8 @@
 package store
 
 import (
+	"antrea.io/nephe/pkg/inventory/indexer"
```
[the remark this answers is not in the document]
Comment (Anandkumar26) [reply pkg/inventory/store/vm_inventory.go:18]: Done

File: pkg/controllers/cloudentityselector/controller_test.go (hunk 4 above)
Comment (Anandkumar26) [reply pkg/controllers/cloudentityselector/controller_test.go:1]: Done

File: pkg/cloudprovider/cloudapi/aws/aws_crd_helpers.go [hunk 9]
```diff
@@ -15,6 +15,7 @@
 package aws
 
 import (
+	"antrea.io/nephe/pkg/labels"
```
[the remark this answers is not in the document]
Comment (Anandkumar26) [reply pkg/cloudprovider/cloudapi/aws/aws_crd_helpers.go]: Done

File: pkg/converter/source/virtualmachine.go (hunk 5 above)
Comment (Anandkumar26) [reply pkg/converter/source/virtualmachine.go]: Done

File: pkg/inventory/store/vpc_inventory.go (hunk 6 above)
Comment (Anandkumar26) [reply pkg/inventory/store/vpc_inventory.go]: Done

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

**Pull request merged**